frontend/src/components/Navbar.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react'
```

Comment:
This file is strongly based off of demos from the Material UI docs (https://mui.com/components/app-bar/)